addons/base_automation/views/base_automation_views.xml

```diff
@@ -133,63 +133,59 @@
-                                                            <t invisible="not (update_field_type == 'many2many' and update_m2m_operation == 'clear') or evaluation_type != 'value'">
-                                                                <span>by clearing it</span>
-                                                            </t>
-                                                            <t invisible="not (update_field_type == 'many2many' and update_m2m_operation == 'add') or evaluation_type != 'value'">
-                                                                <span>by adding</span><field name="resource_ref"/>
-                                                            </t>
-                                                            <t invisible="not (update_field_type == 'many2many' and update_m2m_operation == 'remove') or evaluation_type != 'value'">
-                                                               <span>by removing</span><field name="resource_ref"/>
-                                                            </t>
-                                                            <t invisible="not (update_field_type == 'many2many' and update_m2m_operation == 'set') or evaluation_type != 'value'">
-                                                                <span>by setting it to</span><field name="resource_ref"/>
-                                                             </t>
-                                                            <t invisible="update_field_type == 'many2many' and evaluation_type == 'value'">
-                                                                <span invisible="evaluation_type != 'value'">to</span>
-                                                                <span invisible="evaluation_type != 'equation'">as</span>
-                                                                <field name="resource_ref" invisible="not (value_field_to_show == 'resource_ref' and evaluation_type == 'value')" />
-                                                                <field name="selection_value" invisible="not (value_field_to_show == 'selection_value' and evaluation_type == 'value')" class="d-inline"/>
-                                                                <field name="update_boolean_value" invisible="not (value_field_to_show == 'update_boolean_value' and evaluation_type == 'value')" class="d-inline"/>
-                                                                <em invisible="not (value_field_to_show == 'value' and evaluation_type == 'value')" t-out="record.value.raw_value" class="d-inline"/>
-                                                                <code invisible="not (evaluation_type == 'equation')" t-out="record.value.raw_value"/>
+                                            <t t-name="kanban-card" class="row g-0">
+                                                <div class="d-flex align-items-center gap-1">
```

Comment:
you could probably get rid of this div (by moving some of its classnames to the node above)